packages/experimental/Drawer/android/gradle.properties

```diff
@@ -0,0 +1,6 @@
+FURNDrawer_kotlinVersion=1.5.31
+FURNDrawer_fluentuiVersion=0.0.21
+FURNDrawer_compileSdkVersion=29
+FURNDrawer_buildToolsVersion=29.0.3
+FURNDrawer_targetSdkVersion=29
+FURNDrawer_minSdkVersion=16
```

Comment:
Because I don't actually know, what is the min sdk we support for Office, which probably informs what the min FURN sdk would be? 16 (Android 4.1) seems really low, and I think most things do 21+ (Android 5.0+)

Reply:
ah icic I will check the Office one and change it here

Reply:
I just checked hermes and the minSdk version they have there is 16. https://github.com/facebook/hermes/blob/a725c6e9740ac1153f34dd0b20303df592ef50bd/android/build.gradle

Reply:
It makes sense that hermes might need to support lower, but that's probably not equal to what office supports. I checked react-native-macos (which is what ends up getting used for Android I believe), and it looks like the minsdk is 21:
https://github.com/microsoft/react-native-macos/blob/f88f01a99e0f6fdd33b4e5c6e8ab882aba07b9ef/ReactAndroid/build.gradle#L392

Reply:
(Should preface with I'm not an android dev, just generally aware that minsdks being too low can and have caused problems for me in the past when updating FURN)